### PR TITLE
Added circle exercise for lesson08

### PR DESCRIPTION
**Pull request by esokada**
Thanks as always for your feedback!

### CODE REVIEW

File: students/esokada/lesson08/circle.py [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+@total_ordering
+class Circle(object):
+
+    def __init__(self, radius):
+        self.radius = radius
+
+    @classmethod
+    def from_diameter(cls, diameter):
+        radius = diameter / 2
```
Comment (natasha-aleksandrova) [inline students/esokada/lesson08/circle.py]: this can be removed since you're not using `radius`

File: students/esokada/lesson08/test_circle.py [hunk 2]
```diff
@@ -0,0 +1,74 @@
+from circle import *
```
Comment (natasha-aleksandrova) [inline students/esokada/lesson08/test_circle.py]: avoid using `*` (star) imports; instead `from circle import Circle`

Comment (esokada) [reply students/esokada/lesson08/test_circle.py]: Right! I'll be more careful in the future.

File: students/esokada/lesson08/circle.py (hunk 1 above)
Comment (esokada) [reply students/esokada/lesson08/circle.py]: Thanks for catching this! Must have been left over, I don't know what I was thinking.

**Pull request merged**